src/com/kainos/buildasub/oop/BuildASub.java

```diff
@@ -11,7 +13,7 @@ public static void main(String[] args) throws Exception {
         Boolean isToastRequired = doYouWantToasted();
         String sauce = chooseSauce();
         //initalising the string which will hold the salad order.
-        String saladChoice = "";
+        ArrayList<String> saladChoice = new ArrayList<String>();
```

Comment:
This should probably use the abstract type `List<String> saladChoice = new ArrayList<String>();`